consutil/lib.py

```diff
@@ -27,6 +28,11 @@
 FLOW_KEY = "flow_control"
 DEFAULT_BAUD = "9600"
 
+PREV_REBOOT_CAUSE="/host/reboot-cause/"
```

Comment:
Seems like there are no reference for this constant?

Reply:
Yes. no use I removed it.